include/verilated.h

```diff
@@ -35,6 +35,7 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+#include <array>
```

Comment:
What needs this? Perhaps leftover from before? (Adding headers here can be compile-time expensive.)

Reply:
Yep, missed it.